data/core/get_started.md

```diff
@@ -243,6 +242,38 @@
     - utter_explain_core
     - utter_direct_to_step2
 
+## not new to rasa/bots, explain difference between nlu and core
+
+* how_to_get_started
+    - utter_getstarted
+    - utter_first_bot_with_rasa
+* deny
+    - action_set_onboarding
+    - slot{"onboarding": false}
+    - utter_ask_which_product
+* explain
+    - utter_explain_stack
+    - utter_explain_nlu
+    - utter_explain_core
```

Comment:
That could get quite a text wall. Should we just explain stack and then ask whether the user needs more explanation?

Reply:
i think we need to rethink the whole flow anyways -- see my PR review above